tests/visual/table.spec.js

```diff
@@ -405,4 +405,24 @@ describe('table', function () {
       },
     );
   });
+
+  test('multi page table', function () {
```

Comment:
Whats the output running with master?

It add ellipsis?

Reply:
oh you meant that as a test, you had commented on #1613  😅 , ill correct now and ill post what it does on master

Reply:
You can keep this also

Reply:
![image](https://github.com/user-attachments/assets/1b996313-b2a8-40e5-a114-bbb0a6da1536)

if we remove the `PDFNumber` wrapper in the wordWidth method (AKA current master)